packages/tools/accessibility/src/HtmlTwin/htmlTwinHostComponent.tsx

```diff
@@ -123,18 +130,52 @@ export class HTMLTwinHostComponent extends React.Component<IHTMLTwinHostComponen
                 })
             );
         }
+        if (!this._observersMap.has(scene.onNewTextureAddedObservable)) {
+            this._observersMap.set(
+                scene.onNewTextureAddedObservable,
+                scene.onNewTextureAddedObservable.add((newTexture) => {
+                    if (this._isFullscreenGUI(newTexture)) {
+                        const advTexture = newTexture as AdvancedDynamicTexture;
+                        setTimeout(() => {

```

Comment:
If the texture is already ready (`newTexture.isReady()==true`), `onLoadObservable` won't be called, so you should check for this case.